backend/main/src/Group/Dtos/ContainerDto.ts

```diff
@@ -0,0 +1,15 @@
+import { Container } from "src/Group/Domain/Entities/Container";
+
+export interface IContainerDto {
+	id: string;
+	name: string;
+	foods: object[];
+}
+
+export const containerDtoMapper = (container: Container): IContainerDto => {
+	return {
+		id: container.id.id,
+		name: container.name,
+		foods: container.foods,
+	};
```

Comment:
You should define food DTO to parse the Container object to the DTO object and should set it as a food's type.
The DTO stands for Data Transfer Object. Which means the DTO should be just primitive data. Please imagine that DTO is used outside of this system. In this case, the JS's object information is redundant and not useful.